Apps/W1/HybridGP/app/src/Migration/Vendors/GPVendorMigrator.codeunit.al

```diff
@@ -43,6 +43,274 @@ codeunit 4022 "GP Vendor Migrator"
         MigrateVendorAddresses(GPVendor);
     end;
 
+    local procedure MigrateVendorDetails(GPVendor: Record "GP Vendor"; VendorDataMigrationFacade: Codeunit "Vendor Data Migration Facade")
```

Comment:
I moved the main Vendor migration procedure up where it should be, instead of at near the bottom of this file.